source/native/symbulation_sgp.cc

```diff
@@ -90,13 +93,18 @@ int symbulation_main(int argc, char * argv[])
                 << std::endl;
       world.sym_points_donated = 0.0;
       world.sym_points_earned = 0.0;
-      emp::Ptr<SGPHost> host = world.GetFullPop().back().DynamicCast<SGPHost>();
-      if (host->HasSym()){
+      int i = 0;
+      emp::Ptr<SGPHost> host = world.GetFullPop()[i].DynamicCast<SGPHost>();
+      while (!host->HasSym()){//Not always a high number of hosts that have symbionts
+          i++;
+          host = world.GetFullPop()[i].DynamicCast<SGPHost>();
+      }
           emp::Ptr<SGPSymbiont> symbiont = host->GetSymbionts().back().DynamicCast<SGPSymbiont>();
           CheckSymbiont(*host, *symbiont);
-        }  
+        /*else if (host->GetPoints() > 0){
```

Comment:
Is this code that will be used? If not, let's remove it right away so we don't have dead code lying around